### PR TITLE
Second fix for `test_token_networks.py`

### PR DESCRIPTION
**Pull request by pcppcp**
- also fixes minor typos and missing class members

### CODE REVIEW

File: raiden/transfer/state.py [hunk 1]
```diff
@@ -115,11 +115,12 @@ def __init__(
         self.payment_mapping = PaymentMappingState()
 
     def __repr__(self):
-        return '<ChainState block:{} networks:{} qty_transfers:{} chain_id:{}>'.format(
+        return '<ChainState block:{} networks:{} qty_transfers:{} chain_id:{} at {}>'.format(
```
Comment (hackaugusto) [inline raiden/transfer/state.py]: why are you adding the id of the object in the representation?

File: raiden/tests/integration/long_running/flaky/test_token_networks.py [hunk 2]
```diff
@@ -93,28 +112,24 @@ def test_participant_selection(raiden_network, token_addresses):
...
-    assert all(x() for x in open_channel_views)
 
     chain = raiden_network[-1].raiden.chain
-    max_wait_blocks = 12
+    max_wait_blocks = 40
 
     while (
-        saturated_count(connection_managers, open_channel_views) < len(connection_managers) and
+        saturated_count(connection_managers, registry_address, token_address) <
+        len(connection_managers) and
```
Comment (hackaugusto) [inline raiden/tests/integration/long_running/flaky/test_token_networks.py]: can you please make this a single line

File: raiden/transfer/state.py (hunk 1 above)
Comment (pcppcp) [reply raiden/transfer/state.py]: 1) it's convention
2) it's useful if you need to check identity of the object (i.e. if you by mistake process the same object twice, which I did)

I can remove it ofc if you think the id is redundant...

Comment (hackaugusto) [reply raiden/transfer/state.py]: For the classes inside the modules `state`, `state_change`, and `events` I think it's a bit unecessary, these classes have structural and not referential equality.

Comment (pcppcp) [reply raiden/transfer/state.py]: ok, I've reverted the files

**Pull request merged**